jsk_perception/sample/sample_insta360_air.launch

```diff
@@ -58,7 +58,10 @@
   </group>
 
   <!-- Split insta360 air image into 2 fisheye image -->
-  <node pkg="jsk_perception" name="split_image" type="split_image.py" output="screen">
+  <node name="split_image"
+        pkg="nodelet" type="nodelet"
+        args="standalone jsk_perception/SplitImage"
```

Comment:
Can you load this nodelet to existing nodelet manager, not standalone?

Reply:
Thank you very much for reviewing. 
I updated at [3bdbad4](https://github.com/jsk-ros-pkg/jsk_recognition/pull/2772/commits/3bdbad45df08f192a25b65862ed31a45c23244bb) and [3b3922d](https://github.com/jsk-ros-pkg/jsk_recognition/pull/2772/commits/3b3922d7229f2a08546bee6d582dc2e30ecde801).

I added arg `launch_manager` so that we can load existing nodelet.

Reply:
I tried to rewrite the following throttle parts with loading existing nodelet in the same way but I cannot now ....
https://github.com/jsk-ros-pkg/jsk_recognition/blob/62afb919a40279b16f240a15c1820d818566d024/jsk_perception/sample/sample_insta360_air.launch#L69-L94

Reply:
Sorry, I solved this for myself at [6bd6e14](https://github.com/jsk-ros-pkg/jsk_recognition/pull/2772/commits/6bd6e14ffd15d25d1416dc3d038a00112fd120c2).